kolibri/core/content/management/commands/importcontent.py

```diff
@@ -330,37 +331,48 @@ def _transfer(  # noqa: max-complexity=16
                     filetransfer = transfer.FileCopy(
                         srcpath, dest, cancel_check=self.is_cancelled
                     )
-
-                try:
-                    status = self._start_file_transfer(
-                        f, filetransfer, overall_progress_update
+                    file_transfers.append((f, filetransfer))
+
+            with concurrent.futures.ThreadPoolExecutor(max_workers=5) as executor:
```

Comment:
In future, when we get our multiprocessing compatibility working, we can flexibly use the executor from the `kolibri.core.tasks.compat` module to allow this to be done as a process pool on compatible platforms.

Reply:
Guessing we may also want to allow for the worker pool to be parameterized at some point, but not entirely sure where we would make that live.

Reply:
Note that for downloads specifically, the general consensus I've seen with profiling is that often threads are better than processes, because threads are more lightweight and Python's less optimized switching is sufficient for this case.  So if we did decide to try switching, we should do some profiling first to make sure it's actually a net benefit.

Reply:
Gotcha - yeah, no rush to implement this unless it is actually an improvement!